src/arch/arm/isa/formats/sve_2nd_level.isa

```diff
@@ -245,6 +245,59 @@ namespace Aarch64
         return new Unknown64(machInst);
     }  // decodeSveIntMulAdd
 
+    StaticInstPtr
+    decodeSveMultiplyAccIndexed(ExtMachInst machInst)
+    {
+        RegIndex zda = (RegIndex) (uint8_t) bits(machInst, 4, 0);
+        RegIndex zn = (RegIndex) (uint8_t) bits(machInst, 9, 5);
+        RegIndex zm ;
+
+        //= (RegIndex) (uint8_t) bits(machInst, 19, 16);
+
+        uint8_t size = bits(machInst, 23, 22);
+        uint16_t imm;
+        uint8_t opc = (bits(machInst, 10));
+
+       switch(size) {
+            case 0b00:
+            case 0b01:
+                zm = (RegIndex)(uint8_t)bits(machInst, 18, 16);
+                imm = (uint16_t)(bits(machInst, 22) << 2)
```

Comment:
I undertand zm and imm depend on the element type; may I suggest you to define them within the switch case?
Something like:

       switch(size) {
            case 0b00:
            case 0b01: {
                RegIndex zm = (RegIndex)(uint8_t)bits(machInst, 18, 16);
                RegIndex imm = (uint16_t)(bits(machInst, 22) << 2)

                [...]
            }



I would like to avoid having some code using uninitialized variables by accident